openedx/core/djangoapps/util/storage.py

```diff
@@ -0,0 +1,18 @@
+"""
+Django storage utilities.
+"""
+from django.core.files.storage import get_storage_class
+from django.utils.lru_cache import lru_cache
+
+
+@lru_cache()
+def get_storage(storage_class=None, **kwargs):
```

Comment:
Some questions/comments about this refactor:
1. I don't think you deleted the old code.
2. Why are you moving only part of the original storage.py file?
3. If you really want to move only part, maybe they should have different names?
4. Nit: It would be great if refactors like this could be kept to a separate commit when possible.